### ISSUE TITLE
mdoutliner6 SIGSEGV on v0.7.0

### ISSUE THREAD

**Issue by SnakeTwix** [mappu/miqt#110]
Couldn't find a template for opening issues, so excuse me.

Ran everything in the example folder apart from: `libraries, android, marhsalling, uidesigner, windowsmanifest` and the only thing that didn't work was mdoutliner6. 

 Here's the output during runtime: 
 ```
 panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x94a07d]

goroutine 1 [running, locked to thread]:
github.com/mappu/miqt/qt6.(*QWidget).AddActionWithText.func2(0x111df41?, {0x7?, 0x25c3d0e0?})
        /home/aria/.local/share/go/pkg/mod/github.com/mappu/miqt@v0.7.0/qt6/gen_qwidget.go:1308 +0x1d
github.com/mappu/miqt/qt6.(*QWidget).AddActionWithText(0x0, {0x111df41?, 0x7})
        /home/aria/.local/share/go/pkg/mod/github.com/mappu/miqt@v0.7.0/qt6/gen_qwidget.go:1308 +0x78
main.NewAppWindow()
        /home/aria/Desktop/coding/qt-go/main.go:70 +0xad
main.main()
        /home/aria/Desktop/coding/qt-go/main.go:183 +0x2d
exit status 2
```

Investigated this a little bit,  the issue seems to be on [line 68](https://github.com/mappu/miqt/blob/5f7b9a47b8935942a651a78b2b100146238ff65d/qt/gen_qmenubar.go#L127) when creating the `fileMenu`. It all boils down to passing a `nil` pointer for `h_QWidget` [here](https://github.com/mappu/miqt/blob/5f7b9a47b8935942a651a78b2b100146238ff65d/qt/gen_qmenubar.go#L127). 

Is this example broken for v0.7.0 release or am I doing something wrong?

**Comment by mappu** [comment]
I see the problem. Yes, this example will be broken in v0.7.0, but it's working again in miqt master (b31a1e3dfeba1e403ae05b9ac7c2969f718d3799) since #101.

I'll tag a new v0.7.1 release soon with collected bugfixes.

**Comment by mappu** [comment]
I can confirm the crash on v0.7.0 and confirm the fix on new tag v0.7.1. Please try out v0.7.1 and let me know if it's fixed for you too,
